docs/docs/glossary.md

```diff
@@ -1,6 +1,8 @@
-import HorizontalNavList from "../../www/src/components/horizontal-nav-list.js"
+---
+title: "Glossary"
```

Comment:
Do the quotes matter? I've been recommending removing them.

Reply:
Yeah you're right they don't matter, I can remove 'em

Reply:
```suggestion
title: Glossary
```